Models/User.cs

```diff
@@ -19,5 +19,9 @@ public User(string uname, string password, string accessType)
             Password = password;
             AccessType = accessType.Trim();
         }
+
+        public User()
+        {
+        }
```

Comment:
This is constructor by default, you don't need it, do you?

Reply:
Not now,  but I think I will, and as you told me 
User u = new User();
u.Login = "bla";
u.Password = "blo"
u.AccessType = "blu"

Reply:
![image](https://user-images.githubusercontent.com/37458100/57158403-024fbb80-6dec-11e9-910d-eb2b1b3a0a1a.png)

Reply:
Well, of course
You should delete all c-tors, or write a c-tor that doesn't take arguments

Reply:
then, I won't delete this one